docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -37,3 +37,20 @@ To enable above precondition and onboarding scenario in MVD, the following steps
 7. The Registration Service will be configured with a policy, that requires a GAIA-X membership Verifiable Credential issued by the GAIA-X 
    Authority to verify participant's GAIA-X membership.
 
+### Public key infrastructure
+
+Public keys will be represented in [JSON Web Key](https://www.rfc-editor.org/rfc/rfc7517#section-4) format. 
```

Comment:
nit: Do we also plan to have similar public key form (JSON Web Key) for other entities e.g. Participants, Registry service. ?

Reply:
I think we are going to reuse the same solution and libraries for other scenarios. For two participants we will just have different kind of claims.
also mentioned here: https://github.com/eclipse-dataspaceconnector/MinimumViableDataspace/tree/main/docs/developer/decision-records/2022-04-11-decentralized-identity#sending-ids-requests